docs/gadgets/bpftrace.md

```diff
@@ -0,0 +1,99 @@
+minikube                         kubelet /proc/1645/fd
+minikube                         coredns /etc/hosts
+minikube                         kubelet /sys/fs/cgroup/cpu,cpuacct/kubepods/besteffort/pod26b59ae5-f76b
+minikube                         kubelet /proc/1565/fd
+minikube                         kubelet /sys/fs/cgroup/cpu,cpuacct/kubepods/burstable/podbd495b7643dfc9
+minikube                         kubelet /proc/1894/fd
+minikube                         kubelet /sys/fs/cgroup/devices/kubepods/besteffort
+minikube                         kubelet /sys/fs/cgroup/devices/kubepods/burstable
+minikube                         kubelet /sys/fs/cgroup/devices/kubepods
+minikube                         kubelet /sys/fs/cgroup/cpu,cpuacct/kubepods/besteffort
+minikube                         kubelet /sys/fs/cgroup/cpu,cpuacct/kubepods/burstable
+minikube                         kubelet /sys/fs/cgroup/cpu,cpuacct/kubepods
+minikube                         kubelet /sys/fs/cgroup/misc/kubepods/besteffort
+minikube                         kubelet /sys/fs/cgroup/misc/kubepods/burstable
+minikube                         kubelet /sys/fs/cgroup/misc/kubepods
+minikube                         kubelet /sys/fs/cgroup/systemd/kubepods/besteffort
+minikube                         kubelet /sys/fs/cgroup/systemd/kubepods/burstable
+minikube                         kubelet /sys/fs/cgroup/systemd/kubepods
+minikube                         kubelet /sys/fs/cgroup/memory/kubepods/besteffort
+minikube                         kubelet /sys/fs/cgroup/memory/kubepods/burstable
+minikube                         kubelet /sys/fs/cgroup/memory/kubepods
+...
+
+# Syscall count by program
+$ kubectl gadget bpftrace -e 'tracepoint:raw_syscalls:sys_enter { @[comm] = count(); }'
+
+NODE                             OUTPUT
+minikube-m03                     Attaching 1 probe...
+minikube-m02                     Attaching 1 probe...
+minikube                         Attaching 1 probe...
+^Cminikube-m02
+minikube
+minikube
+minikube-m02
+minikube-m02                     @[LIST_LEGACY]: 1
+minikube-m02                     @[wpa_supplicant]: 1
+minikube-m02                     @[HangWatcher]: 2
+minikube-m02                     @[SharedWorker th]: 2
+minikube-m02                     @[WebRTC_Worker]: 2
+minikube-m03                     @[cri-dockerd]: 7345
+minikube-m03                     @[EMT-0]: 8105
+minikube-m03                     @[containerd-shim]: 8237
+minikube-m03                     @[gadgettracerman]: 8470
+minikube-m03                     @[runc]: 14030
+minikube-m03                     @[EMT-7]: 15583
+minikube-m03                     @[dockerd]: 22937
+minikube-m03                     @[kubelet]: 27455
+...
+```
+
+### Future improvements
+
+We plan to make the bpftrace and Inspektor Gadget more powerful in the future. Please check TODO
+issue that contains more details about the planned features.
```

Comment:
I assume uprobes are not working. This could be added in the list of limitations.

Reply:
Actually they work -> `./kubectl-gadget bpftrace -e 'uretprobe:/host/bin/bash:readline { printf("read a line\n"); }'`  (note the /host prefix for the bin path). I added a comment on the documentation.